### PR TITLE
PKG-5563: Update transformers to v4.44.1

### PR DESCRIPTION
**Pull request by jnoller**
transformers 4.44.1 

**Destination channel:** defaults

### Links

- [PKG-5563](https://anaconda.atlassian.net/browse/PKG-5563) 
- [Upstream repository](https://github.com/huggingface/transformers)
- [Upstream changelog/diff](https://github.com/huggingface/transformers/compare/v4.44.0...v4.44.1)

### Explanation of changes:

- Update transformers to v4.44.1

[PKG-5563]: https://anaconda.atlassian.net/browse/PKG-5563?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by jnoller** [comment]
Depends on a newer version of huggingface accelerate

**Comment by jnoller** [comment]
Confirmed working on Mac OS and Windows with the updated huggingface_accelerate: https://github.com/AnacondaRecipes/accelerate-feedstock/pull/5

**Review by Jrice1317 — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -55,7 +55,8 @@ test:
   requires:
     - pip
     # to satisfy `transformers-cli --help` because it raises a warning of missing pytorch/tensorflow/flax for models.
-    - pytorch
+    - pytorch >=2
+    - blas * openblas     # [osx and x86_64]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: I think `blas * openblas     # [osx and x86_64]` should be moved to `run_constrained`.

**Review by skupr-anaconda — COMMENTED** [review]
Suggested changes in `run`:
`huggingface_hub >=0.23.2,<1`, see https://github.com/huggingface/transformers/blob/v4.44.1/setup.py#L120

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml]: Good point, addressed

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**